mevislab.github.io/content/tutorials/visualization/pathtracer/pathtracerexample2.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Example 6.1: Visualization using Path Tracer"
+date: "2023-12-21"
+status: "open"
+draft: false
+weight: 579
+tags: ["Advanced", "Tutorial", "Visualization", "3D", "Path Tracer"]
+menu: 
+  main:
+    identifier: "pathtracer_example2"
+    title: "Comparison between Volume Rendering and MeVisLab Path Tracer"
+    weight: 579
+    parent: "visualization_example6"
+---
+# Example 6.2: Visualization using Path Tracer
+
+## Introduction
+In this tutorial, we will explain the basics of using the `SoPathTracer` module in MeVisLab. You will learn how to create a scene, assign materials, add light sources, and configure the PathTracer to generate renderings.
+
+## Steps to do
+### Develop your network
+
+Begin by loading the example [medical images](/examples/visualization/example6/Volume_1.mlimage) using the `LocalImage` module. Connect it to a `View2D` to visually inspect its contents.
+
+![MR Image of Knee](/images/tutorials/visualization/pathtracer/V6.2_1.png "MR Image of Knee")
+
+
+Connect your `LocalImage` to the `SoPathTracerVolume` module to initiate volume rendering. Add the `SoPathTracer` module into your SDK and connect both to the `SoExaminerViewer` module to visualize the rendered volume. the `SoPathTracer` module serves as the main rendering component. It offers parameters for resulotion, camera settings, background colors and iteration control.
+
+{{<alert class="info" caption="Additional Info">}}
+It's essential to consistently position the `SoPathTracer `module on the right side of the scene. This strategic placement ensures that the module can render all objects located in the scene before it accurately. 
+{{</alert>}}
+
+![SoPathTracerVolume & SoPathTracer](/images/tutorials/visualization/pathtracer/V6.2_2.png "SoPathTracerVolume & SoPathTracer")
+
+If you check your `SoExaminerViewer` you will see a black box, this indicates often a need for additional adjustments to color, light and transparency.
```

Comment:
Are you sure? Even after adding a light, it remains a box. Its because of the LUT, not because of the light and material